src/main/java/com/stripe/model/issuing/CardPin.java

```diff
@@ -1,75 +0,0 @@
-// Generated by com.stripe.generator.entity.SdkBuilder
```

Comment:
Not entirely sure why this disappeared though I'm not fully familiar with that resource